Healthy/Classes/Utilities/Logger/FileSystemLogger.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+final class FileSystemLogger: Logging {
+    private var logCache: [String] = []
+    private let logFileName = "app.log"
+    private let logQueue = DispatchQueue(label: "", qos: .background, attributes: .concurrent)
```

Comment:
```suggestion
    private let logQueue = DispatchQueue(label: "com.motoon.default-logger", qos: .background, attributes: .concurrent)
```

Dispatch queue label should never be empty.